### ISSUE TITLE
Caps-, Num-, ScrollLock indicator LED's?

### ISSUE THREAD

**Issue by abrahamssonmathias** [kiibohd/controller#225]
Hi! I've read through the documentation and the source code to the best of my abilities, but I can't seem to find where to define the I/O for the lock modifier indicator LED's. Could anyone please point me in the right direction?

**Comment by abrahamssonmathias** [comment]
Should add that I have so far been using the Teensy 3.2 guide on defining the matrix and mapping keys.

**Comment by cookiesowns** [comment]
Also interested in this. As far as I'm currently aware there's no way to have a visual indication that these modifiers are activated.

**Comment by Kody-Quintana** [comment]
Is this still not possible?

**Comment by haata** [comment]
Not quite yet, the next firmware release should include support for indicator LEDs (I've been fixing quite a few bugs related to lock LED lissues).
https://github.com/kiibohd/controller/issues/313

I'll also be tracking this issue here for Kira: https://github.com/kiibohd/controller/issues/314

**Comment by haata** [comment]
Yes! That code already works for turning the LEDs on (just not turning the LEDs off, haha).
